ui/apps/platform/src/routePaths.ts

```diff
@@ -204,6 +206,10 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
     dashboard: {
         resourceAccessRequirements: everyResource([]),
     },
+    'deferral-configuration': {
+        featureFlagDependency: ['ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL'],
+        resourceAccessRequirements: everyResource(['Administration']),
```

Comment:
@md2119 is `'Administration'` the correct resource used to gate access to the new deferral configuration page and APIs?

Reply:
Yes